README.md

```diff
@@ -96,16 +96,16 @@ oneDNN supports platforms based on the following architectures:
 
 The library is optimized for the following CPUs:
 * Intel Atom processor with Intel SSE4.1 support
-* 4th, 5th, 6th, 7th, and 8th generation Intel(R) Core(TM) processor
-* Intel(R) Xeon(R) processor E3, E5, and E7 family (formerly Sandy Bridge,
+* 4th, 5th, 6th, 7th, and 8th generation Intel&reg; Core&trade; processor
+* Intel&reg; Xeon&reg; processor E3, E5, and E7 family (formerly Sandy Bridge,
   Ivy Bridge, Haswell, and Broadwell)
-* Intel(R) Xeon Phi(TM) processor (formerly Knights Landing and Knights Mill)
-* Intel Xeon Scalable processor (formerly Skylake and Cascade Lake)
-* future Intel Xeon Scalable processor (code name Cooper Lake)
+* Intel&reg; Xeon Phi&trade; processor (formerly Knights Landing and Knights Mill)
+* Intel&reg; Xeon&reg; Scalable processor (formerly Skylake and Cascade Lake)
```

Comment:
In this and some other places trademark symbols are not necessary, as the trademark was used with proper symol earlier in the text.

Reply:
:+1: